### DISCUSSION TITLE
batch processing

### DISCUSSION THREAD

**Discussion by aliakbarhamzeh1378** [coqui-ai/STT#2099] [Q&A] [answered]
Hello everyone.
is there any way for batch files processing for inference? is that using GPU or only working on CPU?

**Comment by reuben** [comment, marked as answer]
For GPU accelerated batch inference you can use our [checkpoint inference functions](https://stt.readthedocs.io/en/latest/Checkpoint-Inference.html) in a Python environment that has GPU enabled TensorFlow installed.
